products/models/store.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class Store(models.Model):
+    name = models.CharField(max_length=120)
+    address = models.CharField(max_length=120)
+    description = models.TextField(blank=True, null=True)
+    email = models.EmailField(blank=True, null=True)
+    phone = models.CharField(max_length=20, blank=True, null=True)
+
```

Comment:
same here, pls add relation to the product through model StoreInventory as we did in the lesson between order and product.